nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -26,6 +26,8 @@
 
 [`sudo-rs`]: https://github.com/memorysafety/sudo-rs/
 
+- Users now default to a primary group named the same as the user, and with group id same as user id. Before it was "users" with group id 100 for normal users and "" (empty) for system users -- which meant system users required explicitly setting the primary group. This changes gives better control over permissions/file sharing and aligns NixOS with other distros like Ubuntu/Debian/Fedora/Arch Linux. For backward compatibility, normal users still default to group "users" when `system.stateVersion < 23.11`.
```

Comment:
This is having a merge conflict, you might want to consider putting it at somewhere more arbitrary (order-wise) so it's less likely to get into merge conflicts again.

Reply:
Thanks, I put it "in the middle" now.